vocabularies/borehole-purpose.ttl

```diff
@@ -108,7 +108,7 @@ bhpur:oil-shale a skos:Concept ;
     skos:topConceptOf <http://linked.data.gov.au/def/borehole-purpose> .
 
 bhpur:water a skos:Concept ;
-    skos:definition "Wells and bores drilled under permits governed by the Queensland Water Act 2000 and Petroleum and Gas (Production and Safety) Act 2004"@en ;
+    skos:definition "Wells and bores drilled under permits governed by the Queensland Water Act 2000"@en ;

```

Comment:
There are also water boreholes drilled under (MRA s. 334ZP(5)(b)) and (MRR s. s31A-31C).
See https://www.business.qld.gov.au/industries/mining-energy-water/resources/minerals-coal/reports-notices/water-reporting

Reply:
The conceptScheme is based around the Act(s) that governs the borehole. 
Do we want all water bores classified together at the purpose level, despite the fact that they are governed under separate acts?
Or should we stay true to the concept scheme where a water bore drilled under a mineral permit is subject to MRA conditions and is therefore a Mineral (Act) Borehole with a Water supply purpose. i.e. water bores are grouped at the sub-purpose (functional) level but retain information on the act that governs them at the purpose level.

@LizDerrington @DavidCrosswellGSQ thoughts?

Reply:
```suggestion
    skos:definition "Wells and bores drilled under permits governed by the Queensland Water Act 2000. Additional rights, obligations, and responsibilities may be conferred by intersecting legislation on wells and bores drilled by mineral and coal permit holders and petroleum and gas permit holders under the Mineral Resources Act 1989 and the Petroleum and Gas (Production and Safety) Act 2004 respectively "@en ;

```

Reply:
I'm not fussed on this. We could so "Queensland Water Act 2000 and other relevant Acts".

Reply:
```suggestion
    skos:definition "Wells and bores drilled to access water resources under permits governed by the Queensland Water Act 2000 and other relevant Acts."@en ;

```